src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -214,6 +214,68 @@ struct Input
   }
 };
 
+//!@brief Our allocatorId, based on execution policy.
+static int allocatorId = axom::INVALID_ALLOCATOR_ID;  // Set in main.
+static int allocatorIdForPolicy(quest::MarchingCubesRuntimePolicy policy)
+{
+  //---------------------------------------------------------------------------
+  // Set default allocator for possibly testing on devices
+  //---------------------------------------------------------------------------
+  int aid = axom::INVALID_ALLOCATOR_ID;
+
+  // clang-format off
+  if(policy == axom::quest::MarchingCubesRuntimePolicy::seq)
+  {
+    aid = axom::execution_space<axom::SEQ_EXEC>::allocatorID();
+  }
+#if defined(AXOM_USE_RAJA)
+#ifdef _AXOM_MC_USE_OPENMP
+  else if(policy == axom::quest::MarchingCubesRuntimePolicy::omp)
+  {
+    aid = axom::execution_space<axom::OMP_EXEC>::allocatorID();
+  }
+#endif
+#ifdef _AXOM_MC_USE_CUDA
+  else if(policy == axom::quest::MarchingCubesRuntimePolicy::cuda)
+  {
+    aid = axom::execution_space<axom::CUDA_EXEC<256>>::allocatorID();
+  }
+#endif
+#ifdef _AXOM_MC_USE_HIP
+  else if(policy == axom::quest::MarchingCubesRuntimePolicy::hip)
+  {
+    aid = axom::execution_space<axom::HIP_EXEC<256>>::allocatorID();
+  }
+#endif
+#endif
+  // clang-format on
+
+  SLIC_ERROR_IF(
+    aid == axom::INVALID_ALLOCATOR_ID,
+    axom::fmt::format("Cannot find allocator id for policy '{}'", policy));
+  return aid;
+}
+
+//!@brief Put a conduit::Node array data into the specified memory space.
+template <typename T>
```

Comment:
Conduit reads the mesh file into CPU memory.  Moving it to GPU memory simulates how an application's behavior.